etc/bash/docker.sh

```diff
@@ -1,6 +1,26 @@
 #!/usr/bin/env bash
 
 cd $(dirname $0)/../docker
-docker-compose up -d
-docker-compose exec php bash -c "cd /var/www/taskmanager && composer install"
-docker-compose exec php bash -c "cd /var/www/identityaccess && composer install"
+cp .env.dist .env || echo "Previosuly created env"
```

Comment:
Any possibility to check that .env exists before copying? Right now the `.env` is overwritten even if exists